pkg/controller/backup_sync_controller.go

```diff
@@ -107,4 +126,35 @@ func (c *backupSyncController) run() {
 			}
 		}
 	}
+
+	c.deleteUnused(cloudBackupNames)
+	return
+}
+
+// deleteUnused deletes backups from Kubernetes if there is no corresponding backup in the client
```

Comment:
Do you mean `if there is no corresponding backup in the object storage`?

Reply:
Yes. I'll correct that.